conbench/tests/api/test_results.py

```diff
@@ -791,25 +783,6 @@ def _assert_commit_repo_without_hash(self, response):
         )
         return benchmark_result, new_id
 
-    def test_create_no_commit_context(self, client):
-        self.authenticate(client)
-        data = copy.deepcopy(self.valid_payload)
-        data["run_id"] = _uuid()
-        del data["github"]
-
-        # create benchmark without commit context
-        response = client.post("/api/benchmarks/", json=data)
-        benchmark_result, new_id = self._assert_none_commit(response)
-        location = "http://localhost/api/benchmarks/%s/" % new_id
-        self.assert_201_created(response, _expected_entity(benchmark_result), location)
-
-        # create another benchmark without commit context
-        # (test duplicate key duplicate key -- commit_index)
-        response = client.post("/api/benchmarks/", json=data)
-        benchmark_result, new_id = self._assert_none_commit(response)
-        location = "http://localhost/api/benchmarks/%s/" % new_id
-        self.assert_201_created(response, _expected_entity(benchmark_result), location)
-
```

Comment:
Do we have test case to cover 'github' key validation?

Reply:
Yes, in the parent class of this test class we try to submit an object without each required key and assert that there's an error.